instrumentation/all/lib/opentelemetry/instrumentation/all.rb

```diff
@@ -25,7 +23,9 @@
 require 'opentelemetry-instrumentation-faraday'
 require 'opentelemetry-instrumentation-grape'
 require 'opentelemetry-instrumentation-graphql'
+require 'opentelemetry-instrumentation-gruf'
 require 'opentelemetry-instrumentation-http_client'
+require 'opentelemetry-instrumentation-logger'
```

Comment:
Should the Logger bridge be part of `instrumentation/all`? I didn't realize the difference between a "bridge" and an "instrumentation" until after this had been worked out. We could create a separate directory at a peer level with instrumentation named "bridge(s?)" and put the code there. 

If someone knows of an "instrumentation" definition within OTel documentation, that might help.

I'm sure there's other pros/cons, but this is what I have off the top of my head:

**Pros to having it in instrumentation/all:** 
* Easier for people to adopt if they're already installing their instrumentation via all
* One less CI workflow to create, along with all the other structure required for another grouping (ex. `processor`, `propagator`)


**Cons**: 
* The specs reference a ["Bridge API"](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/), so we could be breaking from that concept by grouping this with instrumentation

Reply:
I think it's ok to include it in all since it has the standard instrumentation structure. Although, instrumentation `present do ... end` that check if the instrumentation is little bit loose in my opinion. Maybe also check if `::OpenTelemetry::SDK::Logs` present?